### PR TITLE
log: Add support to GELF protocol

### PR DESCRIPTION
**Pull request by wpjunior**
Thanks @morpheu by motivation (=

### CODE REVIEW

**Comment by scorphus** [comment]
Good job so far, @wpjunior! Why not squash both commits into a single one? Also, remember to always follow project standards for message formatting (hint: the capital A).

**Comment by wpjunior** [comment]
thanks @scorphus (=

File: log/gelfforwarder.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	if extra != "" {
+		data := map[string]interface{}{}
+		if err := json.Unmarshal([]byte(extra), &data); err != nil {
+			bslog.Warnf("unable to parse gelf extra tags: %s", err)
+		} else {
+			b.extra = json.RawMessage(extra)
+		}
+	}
+	b.writer, err = gelf.NewWriter(host)
+
```
Comment (andrestc) [inline log/gelfforwarder.go]: could you remove this blank line?

File: log/gelfforwarder.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (b *gelfBackend) sendMessage(parts *rawLogParts, appName, processName, container string) {
+
```
Comment (andrestc) [inline log/gelfforwarder.go]: could you remove this blank line?

File: log/gelfforwarder.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		},
+		RawExtra: b.extra,
+	}
+
+	err := b.writer.WriteMessage(msg)
+	if err != nil {
+		bslog.Errorf("[log forwarder] failed to send gelf logs: %s", err)
+		return
+	}
+
```
Comment (andrestc) [inline log/gelfforwarder.go]: could you remove this blank line?

**Review by andrestc — COMMENTED** [review]
looks good, just a few nits (i think there are some other blank lines that could be remove but thats probably personal preference)

**Comment by wpjunior** [comment]
@andrestc I've fixed the blank lines.

**Comment by cezarsa** [comment]
Thanks a log @wpjunior !

**Pull request merged**